README.md

```diff
@@ -66,6 +66,38 @@ module.exports = {
 }
 ```
 
+### Minimizing For Production
+
+While webpack 5 is likely to come with a CSS minimizer built-in, with webpack 4 you need to bring your own. To minify the output, use a plugin like [optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin):
+
+**webpack.config.js**
+
+```js
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+module.exports = {
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: "[name].css",
+      chunkFilename: "[id].css"
+    }),
+    new OptimizeCssAssetsPlugin()
```

Comment:
hmm this shouldn't by at 
```js
optimization: {
  minimizer: [
    new OptimizeCssAssetsPlugin(),
  ]
}
```

Reply:
Maybe! I couldn't find docs for this syntax when I created this issue, and I have things working as written. Is it that simple? Just add it to `optimization.minimizer`? Does doing this remove the built-in one for JS?

Reply:
Yeah it will replace minimizer options, defaults are at https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsDefaulter.js#L265-L281
and apply at https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsApply.js#L334-L338

basic the reason to use `optimization.minimizer` is to don't run optimisation on development mode.

Reply:
Ah, interesting. Is this the preferred way then?

Reply:
yep, also make sure to add the `UglifyJsPlugin` too as the default is overriden.

Reply:
@mike-marcacci friendly ping